test/playwright/pages/explorePage.ts

```diff
@@ -14,7 +14,7 @@ export default class ExplorePage {
     this.mapboxCanvas = this.page.getByLabel('Map', { exact: true });
     this.firstDatasetItem = this.page.getByRole('article');
     this.closeFeatureTourButton = this.page.getByRole('button', { name: 'Close feature tour' });
-    this.presetSelector = this.page.locator('#preset-selector');
+    this.presetSelector = this.page.getByTestId('preset-selector');
```

Comment:
I was having to set this to get the parent for it to run.
```
const toolbar = page.locator('#preset-selector');
```

otherwise, the test would fail for me.  I'd prefer to use your test Id if it is working for you, though.

Reply:
I don't have any issues with using the test id. What exactly is the problem that you are seeing?

Reply:
The tests always fails unless I switch to locating by ID.  I've attached the trace from what I see after syncing with origin and deleting and re-installing node_modules to try minimizing a chance of mismatch.

the only data-testid I see is on the analysis message
<img width="683" alt="Screenshot 2024-10-29 at 9 22 25 AM" src="https://github.com/user-attachments/assets/b4f87916-46d3-4a4f-8311-56e793008d39">

the select has the id, but no data-testid.  
<img width="928" alt="Screenshot 2024-10-29 at 9 23 33 AM" src="https://github.com/user-attachments/assets/925a859a-7a70-4ff8-b633-69fe68c89989">

This is the test run with `yarn playwright test exploreAoi.spec.ts`

[traceAOI.zip](https://github.com/user-attachments/files/17559393/traceAOI.zip)

Reply:
<img width="1394" alt="Screenshot 2024-10-31 at 11 44 12" src="https://github.com/user-attachments/assets/2f9e1794-c88c-4b32-9cbf-2699358d00f2">
This is very strange, the data-testid does show up as expected for me. Can you try to run the whole test suite, instead of only the exploreAoi.spec.ts? Also, I assume you did re-start the local server with `yarn serve` at some point? Maybe somehow the local build isn't updating as it should? Wonder what else it could be that is different for me...

Reply:
I ran the whole test suite locally including running this test individually, and it always passes with a ✅ in my case.

<img width="522" alt="Screenshot 2024-10-31 at 12 12 33" src="https://github.com/user-attachments/assets/48f5d5e8-ad1b-4cc0-b83e-33690169b0ca">